src/clif-util.rs

```diff
@@ -1,5 +1,5 @@
-#![deny(trivial_numeric_casts, unused_extern_crates, unstable_features)]
-#![warn(unused_import_braces)]
+#![deny(trivial_numeric_casts)]
+#![warn(unused_import_braces, unstable_features, unused_extern_crates)]
```

Comment:
This may have been an unintentional merge; please move `unstable_features` and `unused_extern_crates` back to `deny`.